docs/getting-started/gs-faqs.md

```diff
@@ -43,6 +44,16 @@ In addition, in almost all cases, UID2 doesn't store any values at all once the
 
 No. UID2 participants must not generate UID2s from Protected Health Information, as defined by the Health Insurance Portability and Accountability Act (HIPAA), even if they have obtained consent to engage in marketing with respect to such data.
 
+#### Should I use a Public Operator or a Private Operator?
+
+For most participants, <Link href="../ref-info/glossary-uid#gl-public-operator">Public Operator</Link> is the simplest solution. A Public Operator integration is a much easier option than creating your own <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link>. However, having a Private Operator instance has many advantages. The best choice depends on your unique scenario and needs.
```

Comment:
`creating your own private operator` -- should this be `hosting your own private operator`? or something similar (setting up, running, using, etc)? A participant cannot "create" a private operator?

Reply:
Fixed, two instances, thx.